pkg/tests/end_to_end_tests/jaeger_store_integration_tests/trace_compare.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
I also copied this file in my latest PR, I think it's worth keeping https://github.com/timescale/promscale/pull/1678/files#diff-cdda0957b5a9bdfc7d49e834ef343f4850c254f1dfcffc78b2575fe09a7feb3bR1